src/common/depthmap_toolkit/utils.py

```diff
@@ -36,9 +72,8 @@ def convert_2d_to_3d_oriented(intrisics: list, x: float, y: float, z: float) ->
     res = convert2Dto3D(calibration[1], x, y, z)
     if res:
         try:
-            res = point_rotation_by_quaternion(res, rotation)
-            for i in range(0, 2):
-                res[i] = res[i] + position[i]
+            res = [-res[0], -res[1], res[2]]
+            res = matrix_transform_point(res, matrix)
         except NameError:
             i = 0
```

Comment:
Please note that this PEP-8 is failing now because the variable `i` is not used in the try block anymore.
```suggestion
            pass
```